mantle/platform/platform.go

```diff
@@ -476,9 +477,26 @@ func CheckMachine(ctx context.Context, m Machine) error {
 		return fmt.Errorf("not a supported instance: %v", string(out))
 	}
 
+	// check systemd version on host to see if we can use `busctl --json=short`
+	var systemdVer int
+	var failedUnitsCmd string
+	minSystemdVer := 240
+	out, stderr, err = m.SSH("rpm -q --queryformat='%{VERSION}\n' systemd")
```

Comment:
Personally, I tend to prefer using the upstream tools like e.g. `systemctl --version` because it keeps us more independent of RPM and more portable across distributions. I don't think you should change this code, but something to keep in mind in the future.

Another approach is to do feature detection via e.g. `busctl --help` and look for `--json` there.